src/utils/authentication/aad.strategy.ts

```diff
@@ -1,10 +1,4 @@
-import {
-  forwardRef,
-  Inject,
-  Injectable,
-  Logger,
-  UnauthorizedException,
-} from '@nestjs/common';
+import { forwardRef, Inject, Injectable, LoggerService } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config/dist';
```

Comment:
looks odd?

Reply:
I see nothing odd here - this is default Prettier config and refactoring.